client.cpp

```diff
@@ -239,16 +243,12 @@ voltdb::Procedure *prepare_to_invoke(INTERNAL_FUNCTION_PARAMETERS, voltclient_ob
 
     // Check if the client is created.
     if (obj->client == NULL) {
-        zend_throw_exception(zend_exception_get_default(TSRMLS_C), NULL,
-                             voltdb::errConnectException TSRMLS_CC);
-        return NULL;
+        throw voltdb::ConnectException();
```

Comment:
Can't throw C++ exception, should use the old behavior to construct a proper PHP exception.